internal/pkg/build/sources/conveyorPacker_zypper.go

```diff
@@ -469,3 +504,44 @@ func rpmPathCheck() (err error) {
 
 	return nil
 }
+
+/*
+Parse the /etc/os.release file to a a struct, so that SUSE versions
+need not to be set on a SLE syste,
+*/
```

Comment:
Please use `//` style comments to match the rest of the code base.

Also there's a duplicate `a` in the first line.